app/controllers/testcases.js

```diff
@@ -3,40 +3,24 @@
 */
 
 // native modules
-var request = require('request');
+const request = require('request');
 
 // 3rd party modules
-var mongoose = require('mongoose');
 
 // own modules
-var DefaultController = require('./');
+const DefaultController = require('./');
 
-var Controller = function () {
-  var Testcase = mongoose.model('Testcase');
-  var defaultCtrl = new DefaultController(Testcase, 'testcase');
+class TestcasesController extends DefaultController {
+  constructor() { super('Testcase'); }
 
-  this.paramFormat = defaultCtrl.format();
-  this.paramTestcase = defaultCtrl.modelParam();
-
-  this.all = (req, res, next) => {
-    // dummy middleman function..
-    next();
-  };
-
-  this.get = defaultCtrl.get;
-  this.find = defaultCtrl.find;
-  this.create = defaultCtrl.create;
-  this.update = defaultCtrl.update;
-  this.remove = defaultCtrl.remove;
-
-  this.download = (req, res) => {
-    var tc = req.testcase.toObject();
+  download(req, res) {
+    const tc = req.testcase.toObject();
     if (tc.files.length > 0) {
       res.attachment();
-      var url = tc.files[0].href;
+      const url = tc.files[0].href;
       request({
         followAllRedirects: true,
-        url: url,
+        url,
```

Comment:
Eslint by default prefers new Object notation where variables can be initialized to objects with their own name as key, I still agree that mixing the two notation methods in single object looks weird.

See: - new notations in ECMAScript 2015 -
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer